src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/fake_quantize_decomposition_test.cpp

```diff
@@ -0,0 +1,127 @@
+
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "snippets/fake_quantize_decomposition_test.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace ngraph;
+
+namespace {
+
+namespace decompositionInSubgraph {
+const std::vector<TestValues> testValuesDecompositionScalars = {
+    {
+        ov::element::f32,
+        ngraph::Shape{1, 3, 16, 16},
+        ov::element::f32,
+        1.f,
+        {{}, {}, {}, {}},
+    },
+};
+const std::vector<TestValues> testValuesDecompositionPerChannel = {
+    {
+        ov::element::f32,
+        ngraph::Shape{1, 3, 16, 16},
+        ov::element::f32,
+        1.f,
+        {{1, 3, 1, 1}, {1, 3, 1, 1}, {1, 3, 1, 1}, {1, 3, 1, 1}},
+    },
+};
+
+std::vector<std::pair<std::shared_ptr<Node>, std::pair<std::string, std::string> >> operations = {
+    {std::make_shared<opset1::Abs>(), {"Subgraph", "Abs,fakeQuantize"}},
+    {std::make_shared<opset1::Clamp>(), {"Subgraph", "Clamp,fakeQuantize"}},
+    {std::make_shared<opset1::Floor>(), {"Subgraph", "Floor,fakeQuantize"}},
+    {std::make_shared<opset1::Ceiling>(), {"Subgraph", "Ceiling,fakeQuantize"}},
+    {std::make_shared<opset1::Elu>(), {"Subgraph", "Elu,fakeQuantize"}},
+    {std::make_shared<opset1::Erf>(), {"Subgraph", "Erf,fakeQuantize"}},
+    {std::make_shared<opset1::Exp>(), {"Subgraph", "Exp,fakeQuantize"}},
+    {std::make_shared<opset1::LogicalNot>(), {"Subgraph", "LogicalNot,fakeQuantize"}},
+    {std::make_shared<opset1::Negative>(), {"Subgraph", "Negative,fakeQuantize"}},
+    {std::make_shared<opset1::Relu>(), {"Subgraph", "fakeQuantize"}},
+    {std::make_shared<opset5::Round>(), {"Subgraph", "Round,fakeQuantize"}},
+    {std::make_shared<opset1::Sigmoid>(), {"Subgraph", "Sigmoid,fakeQuantize"}},
+    {std::make_shared<opset1::Tanh>(), {"Subgraph", "Tanh,fakeQuantize"}},
+    {std::make_shared<ngraph::op::v0::Gelu>(), {"Subgraph", "Gelu,fakeQuantize"}},
+    {std::make_shared<ngraph::op::v7::Gelu>(), {"Subgraph", "Gelu,fakeQuantize"}},
+    {std::make_shared<ngraph::op::v4::HSwish>(), {"Subgraph", "HSwish,fakeQuantize"}},
```

Comment:
Do we really need to test all this functions? cpu tests are already really heavy